streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -360,11 +371,11 @@ boolean sendingOldValueEnabled() {
         return sendOldValues;
     }
 
-    private void materialize(KTableSource<K, ?> source) {
+    public void materialize(KTableSource<K, ?> source) {
```

Comment:
Unfortunately I need to access this from a separate package in KStreamBuilder.java.